### PR TITLE
fix(pki_test, upgrader.go): Fix the styling errors in both of these files

### PR DESCRIPTION
**Pull request by jchauncey**
On a clean checkout of master if you run `make test-styles` these two files fail that check. This should fix those errors.

---
This change is [Reviewable](https://reviewable.io/reviews/azure/acs-engine/1254)

### CODE REVIEW

**Comment by seanknox** [comment]
Good catch—how these sneak in?

**Comment by jchauncey** [comment]
I have no idea how these got in.

**Comment by seanknox** [comment]
Thanks for fixing the gap in the style checks, too! LGTM.

**Comment by jchauncey** [comment]
this is still inflight while i figure out how to better fix the styling check in circle.

File: scripts/validate-go.sh [hunk 1]
```diff
@@ -19,7 +19,6 @@ exit_code=0
 
 if ! hash gometalinter 2>/dev/null ; then
   go get -u github.com/alecthomas/gometalinter
-  gometalinter --install
```
Comment (seanknox) [inline scripts/validate-go.sh]: I think we need to keep this as first we 1) install gometalinter itself and 2) install the linters themselves.

**Pull request merged**